### ISSUE TITLE
Help/Comment icon should be shown on keyboard focus as well

### ISSUE THREAD

**Issue by blcham** [kbss-cvut/s-forms#319]
![image](https://github.com/kbss-cvut/s-forms/assets/3962237/751adccd-8133-410d-b9ad-c8b4f10a12a9)
